lib/verify/service_provider.rb

```diff
@@ -74,5 +74,9 @@ def handle_response(json_response)
         raise ResponseError.new(parameters["message"], parameters["code"]) if parameters.key?("code")
       end
     end
+
+    def uri_uses_ssl?(uri)
+      uri.port == 443
```

Comment:
This should be `default_port` not `port` to allow for HTTPS on custom ports (e.g. for local SSL testing).